storio-sqlite/src/test/java/com/pushtorefresh/storio/sqlite/integration/ObserveChangesDeleteTest.java

```diff
@@ -0,0 +1,233 @@
+package com.pushtorefresh.storio.sqlite.integration;
+
+import com.pushtorefresh.storio.sqlite.BuildConfig;
+import com.pushtorefresh.storio.sqlite.Changes;
+import com.pushtorefresh.storio.sqlite.operations.delete.DeleteResult;
+import com.pushtorefresh.storio.sqlite.operations.delete.DeleteResults;
+import com.pushtorefresh.storio.sqlite.queries.DeleteQuery;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricGradleTestRunner;
+import org.robolectric.annotation.Config;
+
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Queue;
+
+import rx.Subscription;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(RobolectricGradleTestRunner.class)
+@Config(constants = BuildConfig.class, sdk = 21)
+public class ObserveChangesDeleteTest extends BaseTest {
+
+    // region Delete Object
+
+    @Test
+    public void shouldNotifyAboutDeleteObject() {
+        final Queue<Changes> expectedChanges = new LinkedList<Changes>();
+        expectedChanges.add(Changes.newInstance(UserTableMeta.TABLE));  // Insert
+        expectedChanges.add(Changes.newInstance(UserTableMeta.TABLE));  // Delete
+
+        final EmissionChecker emissionChecker = new EmissionChecker(expectedChanges);
+        final Subscription subscription = emissionChecker.subscribe();
+
+        final User user = putUserBlocking();
+
+        // Insert
+        emissionChecker.awaitNextExpectedValue();
+
+        final DeleteResult deleteResult = deleteUserBlocking(user);
+
+        // Delete
+        emissionChecker.awaitNextExpectedValue();
+
+        assertThat(deleteResult.numberOfRowsDeleted()).isEqualTo(1);
+
+        emissionChecker.assertThatNoExpectedValuesLeft();
+
+        subscription.unsubscribe();
+    }
+
+    @Test
+    public void shouldNotNotifyIfWasNotDeletedObject() {
+        final EmissionChecker emissionChecker = new EmissionChecker(new LinkedList<Changes>());
```

Comment:
If you change `new LinkedList` to `emptyList()` from `Collections` test will be even more obvious and easy to understand!